Upload A File/Download Upload Library.gcomp/Download Upload Library.gcomp

```diff
@@ -16,7 +16,6 @@
 		<CompilerSymbolTable Id="e3cdb554d5454a008effe35b73b648e6" xmlns="http://www.ni.com/PlatformFramework" />
 		<ComponentConfiguration Id="5d5b4cb2ee8641e7a07736a0655d90fc" Name="Configuration1">
 			<CompilerSymbolTable Id="44f082bec2aa42378321b82b6d7e4298" xmlns="http://www.ni.com/PlatformFramework" />
```

Comment:
The change I made here was to unmark the JSLI as exported

![image](https://user-images.githubusercontent.com/1588923/61586171-5b99d200-ab33-11e9-940f-af172ef9dc4c.png)

When the jsli is marked as exported then the nodes in the JSLI are directly accessible from outside the Library.

In this case you already have wrapper gvis for the nodes (I think that's a great practice) so we can prevent a user of the library from accidentally using the JSLI nodes directly by preventing export.